robocat/config/version-config.yaml

```diff
@@ -0,0 +1,13 @@
+versions:
+  pipeline:
+    release: latest
+    bucket: gs://tekton-releases
+    filename: release.yaml
+  triggers: latest
+    release: latest
+    bucket: gs://tekton-releases
+    filename: release.yaml
+  dashboard: latest
+    release: latest
+    bucket: gs://tekton-releases
+    filename: tekton-dashboard-release-readonly.yaml
```

Comment:
this config file looks really useful! i dont see yet where it gets used 🤔

Reply:
It's not used yet :( I removed it for now, in a follow up patch - I though I removed it from here too.
But the idea is to have a manifest like this where we declare everything that we are deploying to a cluster, and then all the cronjobs would be automatically generated from it.

Reply:
makes sense to me! maybe we could exclude it from this commit tho if w're not using it yet? sounds like you did in a follow up patch, does that mean another PR? could we remove it in this PR?

Reply:
Yes, I tried to remove it but failed :P it keeps popping up after I rebase.
I will try again.